src/components/Header/Header.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from '../Navigation/Navigation';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
```

Comment:
also, you can use export here instead default export
```suggestion
export const Header = () => (
```

Reply:
I knew it, but in the task, nothing was said about exporting.
Butб thanks for your advice.